clients/tests/src/main/java/org/projectnessie/client/tests/AbstractSparkTest.java

```diff
@@ -67,12 +64,10 @@ protected void create() throws IOException {
         .set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog");
     spark = SparkSession.builder().master("local[2]").config(conf).getOrCreate();
     spark.sparkContext().setLogLevel("WARN");
-
-    nessieClient = NessieClient.builder().withUri(url).build();
   }
 
-  @AfterAll
-  static void tearDown() throws Exception {
+  @AfterEach
+  protected void tearDown() {
```

Comment:
I don't think we want to stop spark after each run. Why change this?

Reply:
requested in https://github.com/projectnessie/nessie/pull/1783#discussion_r693755112

Reply:
Spark initialization is a bit tricky and stopping it like this could lead to funky issues. It maintains a lot of static state. It also takes a long time to start up so this adds a ton of time to tests.

@snazy we aren't initializing the Spark server in every run, we are initializing it in the first then getting it in the rest (hence `getOrCreate`). If this is particularly bothersome i would `create` in `BeforeAll` and `get` in `BeforeEach`

Reply:
Bummer - missed the `getOrCreate`

Reply:
reverted those changes